lib/puppet/defaults.rb

```diff
@@ -1708,6 +1708,26 @@ def self.default_diffargs
       :default  => "puppet:///pluginfacts",
       :desc     => "Where to retrieve external facts for pluginsync",
     },
+    :pluginsignore => {
```

Comment:
Why move this?

Reply:
Oh, I see. We're sharing this with the two above, but not with the locale sync. I'm not sure how I feel about that - adding `.pot` to this seems sane, but `config.yaml` is such a generic name that I'm not sure we want to do that :/

Reply:
Yeah downloading the config file here really doesn't make sense, since it doesn't have a unique name, and therefore will get overridden with each module with sync. I was worried about the generic name too...

Reply:
I'd still like to see this moved back down to where it was, since it's a "global" download setting, rather than one associated with a particular filesystem mount. Otherwise I'm now :+1: here